js/client/modules/@arangodb/testsuites/rta_makedata.js

```diff
@@ -0,0 +1,145 @@
+    let tests = [
+        fs.join(options.rtasource, 'test_data', 'makedata.js'),
+        fs.join(options.rtasource, 'test_data', 'checkdata.js'),
+        fs.join(options.rtasource, 'test_data', 'checkdata.js'),
+        fs.join(options.rtasource, 'test_data', 'cleardata.js'),
+    ];
+    let count = 0;
+    let counters = { nonAgenciesCount: 1};
+    tests.forEach(file => {
+      count += 1;
+      let args = pu.makeArgs.arangosh(options);
+      args['server.endpoint'] = tu.findEndpoint(options, instanceInfo);
+      args['javascript.execute'] = file;
+      if (options.forceJson) {
+        args['server.force-json'] = true;
+      }
+      if (!options.verbose) {
+        args['log.level'] = 'warning';
+      }
+      if (addArgs !== undefined) {
+        args = Object.assign(args, addArgs);
+      }
+      let argv = toArgv(args);
+      argv = argv.concat(['--', '--minReplicationFactor', '2', '--progress', 'true']);
+      if (options.hasOwnProperty('makedata_args')) {
+        argv = argv.concat(toArgv(options['makedata_args']));
+      }
+      if ((options.cluster) && (count === 3)) {
+        instanceInfo.arangods.forEach(function (oneInstance, i) {
+          if (oneInstance.role === 'dbserver') {
+            stoppedDbServerInstance = oneInstance;
+          }
+        });
+        print('stopping dbserver ' + stoppedDbServerInstance.name +
+              ' ID: ' + stoppedDbServerInstance.id);
+        pu.shutDownOneInstance(options, stoppedDbServerInstance, instanceInfo, counters, false, 10);
+        print("waiting for shutdown...");
+        internal.statusExternal(stoppedDbServerInstance.pid, true);
+        print("gone");
+        argv = argv.concat([ '--disabledDbserverUUID', stoppedDbServerInstance.id]);
+      }
+      require('internal').env.INSTANCEINFO = JSON.stringify(instanceInfo);
+      let rc = pu.executeAndWait(pu.ARANGOSH_BIN, argv, options, 'arangosh', instanceInfo.rootDir, options.coreCheck);
+      res.total++;
+      res.duration += rc.duration;
+      res.status &= rc.status;
+
+      if ((options.cluster) && (count === 3)) {
+        print('relaunching dbserver');
+        pu.restartOneInstance(options, stoppedDbServerInstance, instanceInfo, {});
+      }
+    });
+    return res;
+  }
+  runRtaInArangosh.info = 'runRtaInArangosh';
+
+  let localOptions = _.clone(options);
+  if (localOptions.cluster && localOptions.dbServers < 3) {
+    localOptions.dbServers = 3;
+  }
+  localOptions['server.jwt-secret'] = 'haxxmann';
+
+  let rc = tu.performTests(localOptions, ['rta'], 'rta_makedata_test', runRtaInArangosh);
+  options.cleanup = options.cleanup && localOptions.cleanup;
+  return rc;
+}
+
+
+exports.setup = function (testFns, defaultFns, opts, fnDocs, optionsDoc, allTestPaths) {
+  Object.assign(allTestPaths, testPaths);
+  testFns['rta_makedata'] = makeDataWrapper;
+  opts['rtasource'] = fs.makeAbsolute(fs.join('.', '..','release-test-automation'));
```

Comment:
Is this now hardCoded to `./../release-test-automation`?
Can we make it configureable from the outside please, on my system I do not have the RTA at exactly this position

Reply:
this is the way, the testing.js modules interface accepts sensible defaults.
the array altered by this is the default values array: https://github.com/arangodb/arangodb/blob/devel/js/client/modules/%40arangodb/testutils/testing.js#L164 
which is preset to the actual values here: https://github.com/arangodb/arangodb/blob/devel/js/client/modules/%40arangodb/testutils/testing.js#L588 
and can be overridden from the CLI by specifying `--rtasource somefolder`